content/doc/book/blueocean/activity.adoc

```diff
@@ -13,37 +13,89 @@ wip: true
 
 = Activity View
 
-The Blue Ocean Activity Viewer shows the all activity related to one
+The Blue Ocean Activity View shows the all activity related to one
```

Comment:
@i386 - I switched from "subsection" to "tab", but didn't add "Pipeline" to them.